tests/edge-cases/if-07.txt

```diff
@@ -0,0 +1,2 @@
+brun '(i (q (1 2)) (q (1 2)) (q (1 2)))'
+(q 2)
```

Comment:
Is i intentional to treat a (non-empty) list as `true`?

Reply:
Yes